docker_tippecanoe/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:18.04
+#based on https://hub.docker.com/r/klokantech/tippecanoe
+#with newer ubuntu
+
+ARG BUILD_DATE
+
+LABEL maintainer="wwatkins@usgs.gov"
+LABEL version=0.1.0
+LABEL org.label-schema.build-date=$BUILD_DATE
+
+# Update repos and install dependencies
+RUN apt-get update \
+  && apt-get -y upgrade \
+  && apt-get -y install git build-essential libsqlite3-dev zlib1g-dev wget
+
+#bring in DOI root cert.  Remove this statement for non-USGS persons
+RUN /usr/bin/wget -O /usr/lib/ssl/certs/DOIRootCA.crt http://sslhelp.doi.net/docs/DOIRootCA2.cer && \
+  ln -sf /usr/lib/ssl/certs/DOIRootCA.crt /usr/lib/ssl/certs/`openssl x509 -hash -noout -in /usr/lib/ssl/certs/DOIRootCA.crt`.0 && \
+  echo "\\n\\nca-certificate = /usr/lib/ssl/certs/DOIRootCA.crt" >> /etc/wgetrc; 
+
+# Create a directory and copy in all files
+RUN mkdir -p /tmp/tippecanoe-src
+RUN git clone https://github.com/mapbox/tippecanoe.git /tmp/tippecanoe-src
+WORKDIR /tmp/tippecanoe-src
+
+# Build tippecanoe
+RUN make \
+  && make install
+
+# Remove the temp directory and unneeded packages
+WORKDIR /home/wbeep-processing
+RUN rm -rf /tmp/tippecanoe-src \
```

Comment:
Because this RUN command runs outside of the scope of the other RUN commands, you actually don't save any space. I'd move `apt-get -y remove --purge build-essential && apt-get -y autoremove` up near line 14

I'd also do the following:

```
RUN mkdir -p /tmp/tippecanoe-src

WORKDIR /tmp/tippecanoe-src

RUN git clone https://github.com/mapbox/tippecanoe.git && \
    make && make install && rm -rf /tmp/tippecanoe-src
```
```

Reply:
Ah ok cool.  I just copied that from another Dockerfile